net/wget2/Portfile

```
@@ -0,0 +1,73 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                wget2
+# Make sure to regenerate and replace the man page in the ./files/ directory
+# on update.
+version             2.1.0
+revision            0
+
+homepage            https://gitlab.com/gnuwget/wget2
+
+description         \
+    GNU Wget2 is the successor of GNU Wget, a file and recursive website \
+    downloader.
+
+long_description    {*}${description}.
+
+categories          net www
+license             GPL-3+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+master_sites        https://ftp.gnu.org/gnu/wget
```

Use the GNU fetchgroup instead of hardcoding the URL.

```suggestion
master_sites        gnu:wget
```

---

Changed